language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/util/PerformanceTestUtils.java

```diff
@@ -0,0 +1,52 @@
+/*
+*  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.ballerinalang.langserver.util;
+
+/**
+ * Test utils for running performance tests.
+ */
+public class PerformanceTestUtils {
+
+    /**
+     * Get response time threshold for Completion Performance Test.
+     */
+    public static int getCompletionResponseTimeThreshold() {
+        return Integer.parseInt(System.getProperty("responseTimeThreshold")) / 2;
```

Comment:
Shall we move `"responseTimeThreshold"` to a `private static final String` variable defined in this class.